keyboards/ergodox_ez/led_i2c.c

```diff
@@ -52,6 +52,11 @@ void rgblight_set(void) {
         }
     }
 
+#ifdef RGBW
+    for (uint8_t i = 0; i < RGBLED_NUM; i++) {
+        convert_rgb_to_rgbw(&led[i]);
```

Comment:
Surely we don't need to set `w` to 0 here, if this function is going to calculate it immediately afterwards? (I wonder if GCC is smart enough to optimise that out)

IMO it seems like it'd be more efficient to have `convert_rgb_to_rgbw` take the individual RGB values and return a W value or a `LED_TYPE`, so that the RGB[W] values are only set once per LED. But that may be a job for another day.

Reply:
you're probably right, Should add an "else" there.

And you're probably right. I can change this though, should be simple.